requirements.txt

```diff
@@ -25,23 +23,8 @@ et-xmlfile==1.1.0
 facebook-business==14.0.0
 fastavro==1.5.4
 frozenlist==1.3.1
-gcp-airflow-foundations
+gcp-airflow-foundations>=0.2.8
```

Comment:
why does gcp-airflow-foundations depend on gcp-airflow-foundations lol?

Reply:
i mixed up with eyereturn requirements, i removed it